frontend/lobby/team.tsx

```diff
@@ -56,30 +62,61 @@ class _TeamComponent extends React.PureComponent<Props, State> {
+                <button onClick={this.switchTab} value="team" className={teamView ? 'active' : ''}>Team</button>
+                <button onClick={this.switchTab} className={teamView ? '' : 'active'}>Leaderboards</button>
+            </nav>
+            { teamView &&
+                <div>
+                    <h1>{this.props.teamName}</h1>
+                    <div className="saltines-count">
+                        <h3>Saltines</h3>
+                        <p><img height="20" width="20" src={saltine} alt="Saltine" /><span>{this.props.totalPoints}</span>(current balance)</p>
+                        <p><img height="20" width="20" src={saltine} alt="Saltine" /><span>{this.props.totalPointsAllTime}</span>(earned all-time)</p>
+                    </div>
+                    <div className="rankings-snapshot">
+                        <h3>Rank</h3>
+                        <button onClick={this.switchTab}><sup>#</sup>2<span>in Vancouver</span></button>
+                        <button onClick={this.switchTab}><sup>#</sup>78<span>in Canada</span></button>
+                    </div>
+                    <h3>Your team</h3>
+                    <ul className="team">
+                        <li>{this.props.isTeamAdmin && <button onClick={this.switchTeamAdmin}>X</button>}{this.props.myName} (That's you!) <OnlineIndicator when={this.props.isOnline}/> <AdminIndicator when={this.props.isTeamAdmin}/> </li>
+                        {this.props.otherTeamMembers.map(member =>
+                            <li key={member.id}>{member.name} <OnlineIndicator when={member.online}/> <AdminIndicator when={member.isAdmin}/>{this.props.isTeamAdmin && <button value={member.name} onClick={this.removeTeamMember}>X</button>} </li>
+                        )}
+                        {this.props.otherTeamMembers.length < 4 && <li><button onClick={this.addTeamMemberPrompt}>Add Team Member</button></li>}
+                    </ul>
+                </div>
+            }
+            { !teamView &&
+                <div>
+                    <p>Here be the leaderboards component.</p>
+                </div>
+            }
+            
         </div>;
     }
+
+    @bind private switchTab(event: React.MouseEvent<HTMLButtonElement>) {
+        const tab = event.currentTarget.value
+        tab ? this.setState({ teamView: true }) : this.setState({ teamView: false })
+    }
+
+    @bind private addTeamMemberPrompt(event: React.MouseEvent<HTMLButtonElement>) {
+        confirm('To add someone to your team, give them your team code: ' + this.props.teamCode + '. Your team can have a maximum of 5 people.')
+    }
+
+    private removeTeamMember(event: React.MouseEvent<HTMLButtonElement>) {
+        const name = event.currentTarget.value
+        confirm('Do you want to give ' + name + 'the unceremonious boot?')
+    }
+
+    private switchTeamAdmin(event: React.MouseEvent<HTMLButtonElement>) {
+        confirm('Do you want to remove yourself as the team leader?')
```

Comment:
Do we need to allow team leaders to remove themselves from the team? An easier approach is to add a button that promotes other members to admins, and never allow an option you to remove yourself. That way, if someone wants to remove themselves as admin, they must promote someone else and ask the new admin to do so.

Also FYI currently the code allows multiple admins.

Reply:
Okay great. How about adding an editing team state that will display the remove/make admin buttons?
![edit-team](https://user-images.githubusercontent.com/1463539/39324939-c1d8f524-4945-11e8-83f2-4dd875e6721f.JPG)

When you've marked a player for deletion, it could apply a strikethrough on the person's name. When you've marked a player to be promoted to admin, it places the badge. And then you click Save Changes to confirm?
![editing-team](https://user-images.githubusercontent.com/1463539/39324964-d59d81b0-4945-11e8-8253-8b7b7ac694c3.JPG)

Reply:
Yep, that sounds nice but will be a bit of work to implement :)

Reply:
Great, well in that case for now let's just leave it and worry about the "gameplay" parts since for the first little while it'll just be us testing.